packages/core/content-releases/server/src/routes/release-action.ts

```diff
@@ -0,0 +1,13 @@
+export default {
+  type: 'admin',
+  routes: [
+    {
+      method: 'POST',
+      path: '/release-actions',
+      handler: 'release-action.create',
+      config: {
+        policies: ['admin::isAuthenticatedAdmin'],
```

Comment:
didn't you say you wanted super admin only? I guess we should have a policy for that

Reply:
It will change here: https://github.com/strapi/strapi/pull/18715